### ISSUE TITLE
Toggle Logging On/Off

### ISSUE THREAD

**Issue by vkbellis** [barbeau/gpstest#510]
Consider inserting the word Logging between Status and Map under the main menu with a slider toggle switch to turn logging on and off. Presently, it's not at all clear when any logging is begun or ended.

**Comment by robbitt** [comment]
I agree a logging toggle switch is needed.  Then, one could use the app for GPS mapping of points and linear features.

**Comment by barbeau** [comment]
Not stale

**Comment by barbeau** [comment]
Not stale
